controllers/util/k8sutil.go

```diff
@@ -48,6 +48,9 @@ const (
 	OwnerUIDIndexName = "ownerUID"
 
 	OdfInfoNamespacedNameClaimName = "odfinfo.odf.openshift.io"
+
+	// ExitCodeThatShouldRestartTheProcess is used to restart the process without restarting the entire pod
```

Comment:
Do we really need a comment for this when the name is now self-explanatory?
Also, why all of the spaces before and after each cost? 

```suggestion
```